bundles/org.openhab.binding.bluetooth.roaming/README.md

```diff
@@ -0,0 +1,41 @@
+# Bluetooth Roaming Adapter
+
+This extension adds support for accessing Bluetooth devices from any other configured adapter via a virtual adapter.
+
+## Supported Things
+
+It defines the following bridge type:
+
+| Bridge Type ID | Description                                                                                                |
+|----------------|------------------------------------------------------------------------------------------------------------|
+| roaming        | A virtual Bluetooth adapter that interacts with Bluetooth devices through their nearest Bluetooth adapter. |
+
+## Channels
+
+Devices which use a `roaming` adapter as their bridge also gain the following channels:
+
+| Channel ID       | Item Type | Description                                                     |
+|------------------|-----------|-----------------------------------------------------------------|
+| adapter-uid      | String    | The thingUID of the adapter that is nearest to this device      |
+| adapter-location | String    | The value of the `Location` specified for the nearest adapter    |
+
+## Discovery
+
+A default roaming adapter is discovered automatically, although you can still configure it textually if you want.
```

Comment:
Why? Discovery is meant for real stuff where it is clear that it exists. The roaming adapter is a feature that the user can decide by himself whether he wants to make use of it or not. If he does, he can simply add a roaming bridge very easily.

Reply:
I'll remove it then since users could always create it manually if they wanted.